docs/integrations/web.md

```diff
@@ -36,209 +36,177 @@ There are a couple important concepts to grasp for using Armory. We’ll look at
-
-rule "Welcome new user"
-   when
-       signal: UserConnected() from entry-point "signals"
-       agent: Felix()
-   then
-       agent.showWelcomeScreens();
-       delete(signal);
-end
+We should mention that there are some default standards when it comes to spatial organization of the components on screens: all components apart from the action ones (e.g. buttons taking you to the next screen) gravitate towards top of the screen. The action components are anchored to the bottom of the screen, to avoid screens shifting in size depending on how many components they contain.
+
+You can easily override these standard practices by using the `group()` function, which allows you to create a group of components you want to "stick closer together" on the screen. Check it out:
+```java
+new Screen("customerOnboarding")
+        .add(new Header("logo.png", true))
+        .add(new Title("Tell us about yourself"))
+        .group("center")
+        .add(new Description("Welcome! We would like to get to know you a bit better. Can you start by telling us your name?"))
+        .add(new Input("name", "Type your name here...", "text"))
+        .group("bottom")
+        .add(new SubmitButton("submitName", "Next"))
 ```
-As you can see, there is no need to write out a separate rule for the transition between the `welcome` screen and the `askForName`
-screen - this will already happen because it is specified in agent’s `showWelcomeScreens()`.
 
-Of course, you don’t always want to use predefined sequences of screens (although note that you can just as easily 
-implement slightly more complex condition-based branching in logic, as long as certain actions always lead to the same outcomes). 
-Sometimes you want more flexibility in allowing the system to determine which screen to show to the user depending on the 
-state the user is in.
+This will group the `Description` and `Input` component around the screen center, push the button to the bottom, leaving the `Header` and `Title` by default at the top.
 
-When the screen to show is determined based on other circumstances and not the fact if/which submit action the user made, you can capture this behavior through a rule.
-With Armory, you can easily define multiple screen chains for different stages of the process. This can be beneficial if you want to store some data separately.
-For example, in the welcome screens we asked the user for a name, and we want to store it after the procedure is completed, 
-so we can use it in other screens to add a bit of personalization to the user experience. We will do this inside the `Start user onboarding` rule. 
+The last function we're going to mention here is `setTemplate()`. If you create some specific screen layout you would like to apply to multiple screens (such as content centering, the order of components on the screen), you can create a template and just set it for all screens you want by writing `.setTemplate("TemplateName")` before adding the components:
```

Comment:
This part is unclear. Custom templates are used when there is a lot of custom logic related to the screen.